src/main/java/com/redhat/devtools/lsp4ij/features/formatting/LSPClientSideOnTypeFormattingTypedHandler.java

```diff
@@ -157,13 +156,17 @@ private static Result handleCloseBraceTyped(@NotNull Project project,
                 if ((startOffset > 0) && (documentChars.charAt(startOffset) != openBraceChar)) {
                     startOffset--;
                 }
+                if ((endOffset > 0) && (documentChars.charAt(endOffset) != closeBraceChar) && (documentChars.charAt(endOffset - 1) == closeBraceChar)) {
```

Comment:
These small changes were required to accommodate for some range formatter behavioral differences observed in jdtls when the formatting range includes some additional whitespace beyond the the braced block.